scripts/iam_auto_deployment.sh

```diff
@@ -331,25 +368,18 @@ then
 	$CmdPrefix add-iam-policy-binding $ROOTID \
 	 --member=serviceAccount:$SCRAPINGSA \
 	 --role=roles/storage.admin
-	
+
+	gcloud projects add-iam-policy-binding $PROJECT_ID \
```

Comment:
You also need to add --role=roles/logging.logWriter to the project IAM bindings.

Reply:
fixed